graphs/graph.dart

```diff
@@ -0,0 +1,68 @@
+import '../queue/queue.dart';
```

Comment:
What is this '../queue/queue.dart'? You could use 'dart:collection' but it has different api.

Reply:
Oh! Actually this code required a class Queue which I had on my system as a different file. Now I need to change the whole code. Sorry for the issue. You can close this pull request. I will update you soon.